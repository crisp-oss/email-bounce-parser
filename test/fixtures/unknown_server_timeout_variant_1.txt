This is the mail system at host mailer.acme.email.

I'm sorry to have to inform you that your message could not
be delivered to one or more recipients. It's attached below.

For further assistance, please send mail to postmaster.

If you do so, please include this problem report. You can
delete your own text from the attached returned message.

                   The mail system

<info@payment.com>: conversation with
    _dc-mx.34d953e4r59tt.payment.com[163.234.63.9] timed out while sending
    end of data -- message may be sent more than once
