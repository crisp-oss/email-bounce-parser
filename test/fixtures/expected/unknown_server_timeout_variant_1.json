{
  "bounce": true,
  "email": {
    "body": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system\n\n<info@payment.com>: conversation with _dc-mx.34d953e4r59tt.payment.com[163.234.63.9] timed out while sending end of data -- message may be sent more than once",
    "intro": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system",
    "error": "<info@payment.com>: conversation with _dc-mx.34d953e4r59tt.payment.com[163.234.63.9] timed out while sending end of data -- message may be sent more than once"
  },
  "data": {
    "error": {
      "code": {
        "basic": null,
        "enhanced": null
      },
      "label": "conversation with _dc-mx.34d953e4r59tt.payment.com[163.234.63.9] timed out while sending end of data -- message may be sent more than once",
      "type": null,
      "temporary": false,
      "permanent": false,
      "data": {
        "type": "server_timeout",
        "blocked": false,
        "spam": false
      }
    },
    "recipient": "info@payment.com",
    "server": {
      "hostname": "_dc-mx.34d953e4r59tt.payment.com",
      "ip": "163.234.63.9",
      "port": null
    },
    "command": "DATA"
  }
}